### PR TITLE
*: fix typos about "explicit"

### PR DESCRIPTION
**Pull request by xiekeyi98**
I have checked all "显示" and fix them to "显式".
(Maybe have omissions)

### CODE REVIEW

**Comment by xiekeyi98** [comment]
Before merge this PR,  Please merge https://github.com/pingcap/docs-cn/pull/1091 first.
(First come, first served. 🥇

File: v1.0/sql/date-and-time-types.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ DATETIME 和 TIMESTAMP 的转换：
 
 TIME 转成 DATE 也是类似过程，然后丢弃时间部分。
 
-使用 `CASE()` 函数可以显示的转换类型，比如：
+使用 `CASE()` 函数可以显式的转换类型，比如：
```
Comment (lilin90) [inline v1.0/sql/date-and-time-types.md]:
```suggestion
使用 `CAST()` 函数可以显式地将一个值转换成 DATE 类型，比如：
```

File: v2.0/sql/date-and-time-types.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ DATETIME 和 TIMESTAMP 的转换：
 
 TIME 转成 DATE 也是类似过程，然后丢弃时间部分。
 
-使用 `CASE()` 函数可以显示的转换类型，比如：
+使用 `CASE()` 函数可以显式的转换类型，比如：
```
Comment (lilin90) [inline v2.0/sql/date-and-time-types.md]:
```suggestion
使用 `CAST()` 函数可以显式地将一个值转换成 DATE 类型，比如：
```

File: v2.1/sql/date-and-time-types.md [hunk 3]
```diff
@@ -147,7 +147,7 @@ DATETIME 和 TIMESTAMP 的转换：
 
 TIME 转成 DATE 也是类似过程，然后丢弃时间部分。
 
-使用 `CASE()` 函数可以显示的转换类型，比如：
+使用 `CASE()` 函数可以显式的转换类型，比如：
```
Comment (lilin90) [inline v2.1/sql/date-and-time-types.md]:
```suggestion
使用 `CAST()` 函数可以显式地将一个值转换成 DATE 类型，比如：
```

File: sql/date-and-time-types.md [hunk 4]
```diff
@@ -147,7 +147,7 @@ DATETIME 和 TIMESTAMP 的转换：
 
 TIME 转成 DATE 也是类似过程，然后丢弃时间部分。
 
-使用 `CASE()` 函数可以显示的转换类型，比如：
+使用 `CASE()` 函数可以显式的转换类型，比如：
```
Comment (lilin90) [inline sql/date-and-time-types.md]:
```suggestion
使用 `CAST()` 函数可以显式地将一个值转换成 DATE 类型，比如：
```

File: sql/util.md [hunk 5]
```diff
@@ -93,4 +93,4 @@ dot xx.dot -T png -O
 USE db_name
 ```
 
-切换默认 Database，当 SQL 语句中的表没有显示指定 Database 时，即使用默认 Database。
+切换默认 Database，当 SQL 语句中的表没有显式指定 Database 时，即使用默认 Database。
```
Comment (lilin90) [inline sql/util.md]:
```suggestion
切换默认 Database，当 SQL 语句中的表没有显式指定的 Database 时，即使用默认 Database。
```

Comment (lilin90) [reply sql/util.md]: Grammatically better :)

File: v1.0/sql/util.md [hunk 6]
```diff
@@ -91,4 +91,4 @@ dot xx.dot -T png -O
 USE db_name
 ```
 
-切换默认 Database，当 SQL 语句中的表没有显示指定 Database 时，即使用默认 Database。
+切换默认 Database，当 SQL 语句中的表没有显式指定 Database 时，即使用默认 Database。
```
Comment (lilin90) [inline v1.0/sql/util.md]:
```suggestion
切换默认 Database，当 SQL 语句中的表没有显式指定的 Database 时，即使用默认 Database。
```

File: v2.0/sql/util.md [hunk 7]
```diff
@@ -91,4 +91,4 @@ dot xx.dot -T png -O
 USE db_name
 ```
 
-切换默认 Database，当 SQL 语句中的表没有显示指定 Database 时，即使用默认 Database。
+切换默认 Database，当 SQL 语句中的表没有显式指定 Database 时，即使用默认 Database。
```
Comment (lilin90) [inline v2.0/sql/util.md]:
```suggestion
切换默认 Database，当 SQL 语句中的表没有显式指定的 Database 时，即使用默认 Database。
```

File: v2.1/sql/util.md [hunk 8]
```diff
@@ -93,4 +93,4 @@ dot xx.dot -T png -O
 USE db_name
 ```
 
-切换默认 Database，当 SQL 语句中的表没有显示指定 Database 时，即使用默认 Database。
+切换默认 Database，当 SQL 语句中的表没有显式指定 Database 时，即使用默认 Database。
```
Comment (lilin90) [inline v2.1/sql/util.md]:
```suggestion
切换默认 Database，当 SQL 语句中的表没有显式指定的 Database 时，即使用默认 Database。
```

**Pull request merged**